pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 
 dependencies:
   charcode: ^1.1.0
+  meta: ^1.1.5
```

Comment:
Make this constraint as wide as possible while still containing `@alwaysThrows`.

Reply:
done.